cmd/run.go

```diff
@@ -199,6 +204,16 @@ To skip bundle verification, use the --skip-verify flag.
 	addBundleVerificationSkipFlag(runCommand.Flags(), &cmdParams.skipBundleVerify, false)
 	addBundleVerificationExcludeFilesFlag(runCommand.Flags(), &cmdParams.excludeVerifyFiles)
 
+	// distributed tracing config
+	runCommand.Flags().BoolVarP(&cmdParams.rt.DistributedTracingEnabled, "distributed-tracing", "", false, "enable distributed tracing using OpenTelemetry Tracing")
+	runCommand.Flags().StringVarP(&cmdParams.rt.DistributedTracingAddress, "distributed-tracing-address", "", "localhost:4317", "address of the OpenTelemetry Collector gRPC endpoint")
+	runCommand.Flags().StringVarP(&cmdParams.rt.DistributedTracingServiceName, "distributed-tracing-service-name", "", "opa", "logical name of the service used by OpenTelemetry Tracing")
+	runCommand.Flags().IntVar(&cmdParams.rt.DistributedTracingSampleRatePercentage, "distributed-tracing-sample-rate", 100, "percentage of traces that are sampled and exported")
+	runCommand.Flags().VarP(cmdParams.distributedTracingTLS, "distributed-tracing-tls", "", "set distributed tracing tls scheme")
+	runCommand.Flags().BoolVarP(&cmdParams.rt.DistributedTracingEncryptionSkipVerify, "distributed-tracing-tls-skip-verify", "", false, "disable certificate chain verification")
+	runCommand.Flags().StringVarP(&cmdParams.distributedTracingTLSCertFile, "distributed-tracing-tls-cert-file", "", "", "set path of TLS certificate file of the tracing exporter")
+	runCommand.Flags().StringVarP(&cmdParams.distributedTracingTLSPrivateKeyFile, "distributed-tracing-tls-private-key-file", "", "", "set path of TLS private key file of the tracing exporter")
+
```

Comment:
As @anderseknert mentioned, it would be nice to put these into the yaml/json config instead. I'm trying to find a few pointers on what to do where 👀